### PR TITLE
India/UDISE - Number of Schools having Computer Available by School Category and Management 

### PR DESCRIPTION
**Pull request by thejeshgn**
@ajaits @pradh 
1. Need to import [UDISE Geography ](https://github.com/datacommonsorg/data/tree/master/scripts/india_udise/udise_geography) first. This dataset uses `udiseCode` to resolve location.
2. Raw data is not committed. But raw data is [here](https://github.com/thejeshgn/udise-report-data-downloader/tree/master/raw/). Can be re-downloaded if required
3. [PR](https://github.com/datacommonsorg/schema/pull/207) for Schema .

### CODE REVIEW

File: scripts/india_udise/udise_schools_having_computer_available/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# UDISE+ - Number of Schools having Computer Available by School Category and Management 
+
+## About the Dataset
+This dataset gives the number of schools that have functional toilet facility. This dataset (Report Id:3109) has count of schools by school level and school management.
```
Comment (ajaits) [inline scripts/india_udise/udise_schools_having_computer_available/README.md:4]: s/functional toilet facility/computer facility/

File: scripts/india_udise/udise_schools_having_computer_available/preprocess.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+from india_udise.common.base_school_data import BASE_DATA_FOLDER
+
+# module_dir_ is the path to where this test is running from.
+module_dir_ = os.path.dirname(__file__)
+
+
+class UDISESchoolsHavingComputerAvailable(UDISEIndiaSchoolDataLoaderBase):
+
+    def _get_base_name(self, data_row):
+        name = "Count_School_HasComputerAvailable"
```
Comment (ajaits) [inline scripts/india_udise/udise_schools_having_computer_available/preprocess.py:37]: Can we drop the 'Available' suffix to keep with the practice of using the property value 'Computer' in the name?

File: scripts/india_udise/udise_schools_having_computer_available/preprocess.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        return name
+
+    def _get_base_constraints(self, data_row):
+        constraints = []
+        constraints.append("amenity: dcs:Computer")
```
Comment (ajaits) [inline scripts/india_udise/udise_schools_having_computer_available/preprocess.py:42]: Can you pls add me to the PR for the schema for adding property 'amenity' and class 'Computer' as well?

File: scripts/india_udise/udise_schools_having_computer_available/preprocess.py (hunk 2 above)
Comment (thejeshgn) [reply scripts/india_udise/udise_schools_having_computer_available/preprocess.py]: Done

File: scripts/india_udise/udise_schools_having_computer_available/preprocess.py (hunk 3 above)
Comment (thejeshgn) [reply scripts/india_udise/udise_schools_having_computer_available/preprocess.py]: - Changed `amenity` to `amenityFeature`, existing property. 
- Will tag you in the schema PR.

**Comment by thejeshgn** [comment]
@ajaits and @pradh this is ready.

File: scripts/india_udise/udise_schools_having_computer_available/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+| cat7   | SS (VI-X) - Secondary/Sr. Sec.with grades 6 to 10            |
+| cat8   | SS (IX-X) - Secondary/Sr. Sec.only with grades 9 & 10        |
+| cat10  | HSS (IX-XII) - Higher Secondary with grades 9 to 12          |
+| cat11  | HSS (XI-XII) - Hr.Sec. /Jr. College only with grades 11 & 12 |
+| total  | Total across all the school levels |
+| sch_mgmt_name  | School Management Type |
+
+
+#### Cleaned Data
+Cleaned data will be inside [UDISEIndia_Schools_Having_Computer_Available.zip](UDISEIndia_Schools_Having_Computer_Available.zip) as a CSV file with the following columns.
```
Comment (ajaits) [inline scripts/india_udise/udise_schools_having_computer_available/README.md:33]: The PR has UDISEIndia_Schools_Having_Computer_Available.csv that is 30MB. Is that supposed to be stored as a zip?

Comment (thejeshgn) [reply scripts/india_udise/udise_schools_having_computer_available/README.md:33]: Yes. Done.

**Comment by thejeshgn** [comment]
@ajaits I have removed the CSV and I have pushed the zipped content.

**Comment by thejeshgn** [comment]
Also cross checked one variable - Count_School_HasComputer. Those falls are real.

Count_School_HasComputer

**Review by beets — APPROVED** [review]

**Pull request merged**